.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,56 @@
+name: Bug report
+description: Report an unexpected behaviour with the editor
+labels: bug
+title: "[BUG] "
+body:
+
+    - type: markdown
+      attributes:
+        value: |
+            Please make sure to [search issues](https://github.com/mawww/kakoune/issues?q=is%3Aissue) before creating a new one, to avoid duplicates and centralise conversations.
+
+            If you need to have a question about the editor itself (options, behaviour, code…) answered, make sure to check the [documentation](https://github.com/mawww/kakoune/tree/master/doc/pages) (also available via the `doc` command in the editor) or drop by [IRC](https://web.libera.chat/?channels=#kakoune) to get instant feedback.
+
+    - type: input
+      attributes:
+          label: "Version of Kakoune"
+          description: "If unsure, use the output of command: `kak -version`"
+          placeholder: "v2020.12.31"
+      validations:
+          required: true
+
+    - type: textarea
```

Comment:
I have a shortcut in my browser to open the current text field in my editor.
With the new template that contains 4 text fields I need more roundtrips,
which is mildly annoying.

Anyway, this is really not important because it's not needed very often
(also https://xkcd.com/1172/)